mkdocs.yml

```diff
@@ -116,12 +112,12 @@ plugins:
             show_source: true
             signature_crossrefs: true
           import:
-            # TODO(cookiecutter): You might want to add other external references here
             # See https://mkdocstrings.github.io/python/usage/#import for details
             - https://docs.python.org/3/objects.inv
             - https://frequenz-floss.github.io/frequenz-channels-python/v0.16/objects.inv
             - https://frequenz-floss.github.io/frequenz-sdk-python/v0.25/objects.inv
```

Comment:
Side note: Oh, dammit, we need to depend on the SDK for the actor, but the SDK will depend on the actor too eventually, we need to address https://github.com/frequenz-floss/frequenz-sdk-python/issues/851 sooner than I expected.